Stripe/StripeError+Private.h

```diff
@@ -0,0 +1,15 @@
+//
+//  StripeError+Private.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/22/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSError (STPCustomerContext)
```

Comment:
Ah yeah, I can rename the category to `(StripePrivate)`. There aren't really any other internal-only error methods to move over right now – there are a couple in `STPCheckoutAPIClient`, but it doesn't seem worth moving those if we're going to be removing the class.